src/components/Footer.jsx

```diff
@@ -49,7 +53,8 @@ function Footer() {
             <a
               href="#!"
               type="button"
-              className="rounded-full border-2 border-white text-white leading-normal uppercase hover:bg-black hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out w-9 h-9 m-1">
+              className="rounded-full border-2 border-white text-white leading-normal uppercase hover:bg-black hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out w-9 h-9 m-1"
+            >
```

Comment:
Same and so on, not going to comment on all of these.

Reply:
All of those tags are closing tags for the <a> anchor element